### ISSUE TITLE
Use libvirt network section to specify the apps domain instead using dnsmasq

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#372]
With latest version of libvirt 5.6.0+ it is possible to use dnsmasq options directly into the network section. 
- https://libvirt.org/formatnetwork.html#elementsNamespaces
- https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#console-doesnt-come-up

**Comment by cfergeau** [comment]
This should do the trick for anything runningin the cluster. However, if you want to resolve these names from the host, you'll need to add 192.168.126.1 as a nameserver in /etc/resolv.conf somehow

**Comment by praveenkumar** [comment]
Yes agreed, Also we might not need `apps-crc.testing` instead can use `apps.crc.testing` but that need some discussion, should we change that or not.

**Comment by cfergeau** [comment]
Being able to use `apps.crc.testing` would be nice, since it's more consistent with most clusters.

**Comment by cfergeau** [comment]
RHEL7 and Debian stable do not have new enough libvirt for this (respectively libvirt 4.5.0 and libvirt 5.0.0)
